plan/new_plan_test.go

```diff
@@ -258,6 +258,14 @@ func (s *testPlanSuite) TestCBO(c *C) {
 		best string
 	}{
 		{
+			sql:  "select * from t t1 where 1 = 0",
+			best: "Dummy->Projection",
+		},
+		{
+			sql:  "select count(*) from t t1 having 1 = 0",
```

Comment:
This will return 0 or empty set?
MySQL will return empty set:
mysql> select count(*) from t t1 having 1 = 0;
Empty set (0.00 sec)